package.json

```diff
@@ -16,6 +16,7 @@
     "changelog": "lerna-changelog",
     "clean": "lerna clean",
     "start": "lerna exec --parallel -- npm run start",
+    "test": "cd library && npm test",
```

Comment:
and maybe `&& cd ..` to go back

Reply:
Command concatenation has no effect on the host.

Reply:
cool, didn't know that :D ignore this comment then